docs/developers/README.md

```diff
@@ -172,15 +172,16 @@ Here is an example of a directory layout for a simple Task with 2 script steps:
 |-- run
     `-- 0
         `-- out
+        `-- status
+            `-- exitCode
 |-- scripts
 |   |-- script-0-t4jd8
 |   `-- script-1-4pjwp
 |-- steps
-|   |-- 0 -> /tekton/steps/step-unnamed-0
-|   |-- 1 -> /tekton/steps/step-foo
-|   |-- step-foo
-|   `-- step-unnamed-0
-|       `-- exitCode
+|   |-- 0 -> /tekton/run/0/status
+|   |-- 1 -> /tekton/run/1/status
+|   |-- step-foo -> /tekton/run/1/status
```

Comment:
In https://github.com/tektoncd/pipeline/pull/4416 I have updated tekton debug scripts to write the exitCode to the $(postfile).breakpointexit file. Created https://github.com/tektoncd/pipeline/issues/4422 to track the necessary changes I'll have to make after this PR gets merged.